eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -92,12 +95,19 @@ public ResponseEntity getVariantsByRegion(@PathVariable("regionId") String regio
                                                       annotationVepVersion,
                                               @RequestParam(name = "annot-vep-cache-version", required = false) String
                                                       annotationVepCacheVersion,
-                                              Pageable pageable,
+                                              @RequestParam(value = "page", required = false) Integer page,
+                                              @RequestParam(value = "size", required = false) Integer size,
```

Comment:
looking at the other parameters, size might get confused with region size, so I suggest renaming to pageSize

Reply:
Fixed :)

Reply:
probably renaming `page` to `pageNumber` could also be a good idea to be consistent

Reply:
I have renamed `page` to `pageNumber` :+1: